File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/tank_drive/RedTankDrive.java
package org.firstinspires.ftc.teamcode.teleops.tank_drive;

import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.teamcode.Side;

@TeleOp(name = "Red Tank Drive", group = "Competition")
public class RedTankDrive extends TankDriveBase {
    @Override
    protected Side getSide() {
        return Side.RED;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/tank_drive/RedTankDrive.kt
package org.firstinspires.ftc.teamcode.teleops.tank_drive

import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import org.firstinspires.ftc.teamcode.utils.Side

@TeleOp(name = "Red Tank Drive")
class RedTankDrive : TankDriveBase() {
    override val side: Side = Side.RED
}